### PR TITLE
Add a guide to using node modules in boop

### PR DESCRIPTION
**Pull request by Bachmann1234**
I am not sure if this is the kind of thing you want or not. However, I thought I would share in case you do.

I love Boop (thanks for this btw). However there is one script I really wanted https://curl.trillworks.com/

However, since Boop is written in Swift es6 modules are a no go. So I spent an afternoon hacking around and eventually landed on this trick.

woot

### CODE REVIEW

**Comment by Bachmann1234** [comment]
Credit to this SO for telling me about browserify 
https://stackoverflow.com/questions/22710887/compile-an-npm-module-into-a-single-file-without-dependencies

Credit to this guide for connecting the final dots
https://blog.risingstack.com/running-node-modules-in-your-ios-project/

File: Boop/Documentation/Readme.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Hey there! Thanks for trying out Boop. This documentation should hopefully help
  - [Custom Scripts](CustomScripts.md)
  - [Modules](Modules.md)
  - [Debugging Scripts](Debugging.md)
+ - [Converting Node Modules](ConcertingNodeModules.md)
```
Comment (mrbenshef) [inline Boop/Documentation/Readme.md:10]: Typo in file name

Comment (Bachmann1234) [reply Boop/Documentation/Readme.md]: whoops, fixed!

**Comment by IvanMathy** [comment]
Looks great, thank you so much for the contribution!!

**Pull request merged**